clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -970,6 +970,13 @@ public ClientResponse completed(AbstractResponse response, long timeMs) {
         public ClientResponse disconnected(long timeMs) {
             return new ClientResponse(header, callback, destination, createdTimeMs, timeMs, true, null, null);
         }
+        
+        @Override
+        public String toString() {
+            return "InFlightRequest{" + "header=" + header + ", destination='" + destination + '\'' + ", callback=" + callback
+                    + ", expectResponse=" + expectResponse + ", request=" + request + ", isInternalRequest=" + isInternalRequest
+                    + ", send=" + send + ", sendTimeMs=" + sendTimeMs + ", createdTimeMs=" + createdTimeMs + '}';
```

Comment:
We typically use `(` instead of `{` for `toString` (although there is some inconsistency.